src/uiSelectSortDirective.js

```diff
@@ -31,6 +31,10 @@ uis.directive('uiSelectSort', ['$timeout', 'uiSelectConfig', 'uiSelectMinErr', f
         }
       });
 
+      if (element.data('disallowDrop')) {
```

Comment:
Is this needed? as I don't see anything in the code that is looking for `ondrop`?